src/governor.js

```diff
@@ -1355,7 +1355,7 @@ export const gov_tasks = {
                 }
                 let resSorted = Object.keys(tc.s).sort(function(a,b){return tc.s[b]-tc.s[a]});
                 for (let i=0; i<resSorted.length; i++){
-                    if (global.resource[resSorted[i]].display && atomic_mass[resSorted[i]]){
+                    if (global.resource[resSorted[i]] && global.resource[resSorted[i]].display && atomic_mass[resSorted[i]]){
```

Comment:
there shoudln't be any compatibility issues with this, right?
```suggestion
                    if (global.resource[resSorted[i]]?.display && atomic_mass[resSorted[i]]) {
```
same for the other places

Reply:
According to https://caniuse.com/mdn-javascript_operators_optional_chaining, it looks like this feature is safe to use here.

Reply:
This would be the first usage of the optional chaining operator in Evolve's source code. It's easier to read, but I don't know what our compatibility target is.